gateway/src/apicast/cli/template.lua

```diff
@@ -77,27 +77,42 @@ local function nginx_prefix()
     if match then return match[1] end
 end
 
+local function dirtree(dir, cache)
+  local cached = cache[dir]
+
+  if cached then
+    return pairs(cached)
+  else
+    cache[dir] = {}
+    return fs(dir)
+  end
+end
+
 function _M:interpret(str)
     local interpreter = build_interpreter(str)
 
     local context = self.context
     local filesystem = self.filesystem
     local filter_set = FilterSet:new()
-    local resource_limit = ResourceLimit:new(nil, 1000, nil)
+    local resource_limit = ResourceLimit:new(100000000, 100000, nil)
```

Comment:
Did you find a problem with the previous limits?

Reply:
Yep, including files was hitting some of the limits. Maybe I should set something smaller and more reasonable. These are really ridiculously high.

Reply:
In the end it is fine with the limits we had before.